### ISSUE TITLE
Add an option to turn off zooming to intersected features

### ISSUE THREAD

**Issue by busbus** [Esri/information-lookup#36]
Currently if you have a large intersecting feature, you may want to turn off zoom level.

**Comment by MikeMillerGIS** [comment]
If the user specifies 0, turn off zooming

**Comment by pLeBlanc93** [comment]
Since 0 is a [valid zoom level](http://services.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer), can this be changed to a negative number?

There are three different zoom scenarios I can think of:
* always zoom to a specific scale after clicking (ranges from 0-23)
* "zoom to layer" on intersecting feature (-1?)
   * what if it's not intersecting?
* never zoom at all (-2?)

**Comment by busbus** [comment]
I agree that 0 is a valid zoom level - but -2 may not be intuitive for the user?  Mike - can we place some hints in the configuration?
